components/omega/src/ocn/Tracers.cpp

```diff
@@ -500,6 +486,11 @@ I4 Tracers::exchangeHalo(const I4 TimeLevel) {
 //---------------------------------------------------------------------------
 I4 Tracers::updateTimeLevels() {
 
+   if (NTimeLevels == 1) {
+      LOG_ERROR("Tracers: can't update time levels for NTimeLevels == 1");
+      return -1;
+   }
+
    // Exchange halo
    exchangeHalo(0);
```

Comment:
Now, a time level of 1 in Tracers refers to the next time level, and 0 refers to the current time level. This matches the time level convention of OceanState.
```suggestion
   exchangeHalo(1);
```

Reply:
Thanks for catching this!